rapids/test_rapids.py

```diff
@@ -107,7 +109,7 @@ def test_rapids_spark(self, configuration, machine_suffixes, accelerator):
     # Only need to do this once
     self.verify_spark_job()
 
-  @parameterized.parameters(("STANDARD", ["w-0"], GPU_T4, "12.4"))
+  @parameterized.parameters(("STANDARD", ["w-0"], GPU_T4, "11.8"))
```

Comment:
Is there a reason this switched from 12.4 to 11.8?

Reply:
the default is 12.4, this function is intended to exercise non-default versions of cuda